### PR TITLE
ARCHIE-2715: theming article components for CIO and CCO

### PR DESCRIPTION
**Pull request by ttornese**

### CODE REVIEW

File: src/components/Articles/FinePrint/FinePrint.stories.js [hunk 1]
```diff
@@ -1,50 +1,40 @@
...
-import styled, { ThemeProvider } from 'styled-components';
 
 import FinePrint from './index';
+import { siteKey } from '../../../config/argTypes';
+import { addThemedWrapper } from '../../../config/decorators';
 
 export default {
   title: 'Components/Articles/FinePrint',
   component: FinePrint,
+  decorators: [ addThemedWarpper() ],
```
Comment (jbonAA) [inline src/components/Articles/FinePrint/FinePrint.stories.js:10]: Think this should be `addThemedWrapper()`

**Review by jbonAA — APPROVED** [review]

File: .storybook/main.js [hunk 2]
```diff
@@ -9,6 +9,6 @@ module.exports = {
     '@storybook/addon-viewport/register',
     '@whitespace/storybook-addon-html',
     '@storybook/addon-notes/register',
-    '@storybook/addon-controls',
+    '@storybook/addon-essentials',
```
Comment (meggles) [inline .storybook/main.js:12]: Is Essentials a pack of add-ons that includes Controls?

**Review by meggles — APPROVED** [review]
👍

File: .storybook/main.js (hunk 2 above)
Comment (ttornese) [reply .storybook/main.js:12]: Hi! Yes it does include Controls and a few other addons Storybook highly recommends you can see the list here: https://storybook.js.org/docs/react/essentials/introduction

**Pull request merged**